packages/create-redwood-app/package.json

```diff
@@ -14,8 +15,7 @@
   ],
   "scripts": {
     "build": "yarn node ./build.mjs",
-    "build:watch": "nodemon --watch src --ignore dist,template --exec \"yarn build\"",
-    "prepublishOnly": "NODE_ENV=production yarn build"
+    "test": "yarn node --test-reporter spec ./tests/template.test.js"
```

Comment:
I set `--test-reporter spec` because I was getting the tap reporter on my vscode terminal. I don't really think it'll make that much difference to switch it as we're not passing these test logs into anything programmatic?

https://nodejs.org/api/test.html#test-reporters